scripts/README.md

```diff
@@ -0,0 +1,20 @@
+# Shotgun Pipeline Toolkit Core API
+
+## How to upgrade pyyaml
+
+This package is a vendor widely used by `sgtk` to parse configuration files
+in YAML format.
+
+This package is shipped in source format, that means that only `*.py` are
+included in `python/tank_vendor/yaml` for python versions 2.7 and 3.
+
+If you need to upgrade this package you can use the script `upgrade_pyyaml.py`.
+
+```shell
+cd tk-core/scripts
+python upgrade_pyyaml.py
+```
+
+After running the script the old `python/tank_vendor/yaml` will be moved to `python/tank_vendor/yaml.old`.
```

Comment:
It seems to me like there's no real need to keep the old copy around. If somebody wants to restore it, they'll just revert their local changes.

Why did you decide to keep it around?